### PR TITLE
tests: test search views

### PR DESCRIPTION
**Pull request by jacquerie**
Assets building has been producing an error for quite a while: https://travis-ci.org/inspirehep/inspire-next/builds/116986936#L780. This PR contains a reproducible test case for triggering failures that appears related to it: https://travis-ci.org/inspirehep/inspire-next/builds/116986936#L1434 (but I also see it locally). If someone manages to fix it I will cherry-pick their commit and add it as first commit of this PR.

### CODE REVIEW

**Comment by jacquerie** [comment]
Uh, [this](https://github.com/inspirehep/inspire-next/commit/cf92eb996c15020d63d314d19330b8324276e8f0) looks related.

**Comment by jacquerie** [comment]
@ioannistsanaktsidis, @Panos512, what's the fix for this you have locally?

**Comment by ioannistsanaktsidis** [comment]
@jacquerie you have to install these 2 https://github.com/inspirehep/inspirehep-js https://github.com/inspirehep/inspirehep-search-js together with inspire-next.

File: inspirehep/modules/search/views.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def format_sortoptions(sort_options):
     """Create sort options JSON dump for Invenio-Search-JS."""
     return json.dumps(dict(
         options=sorted_options(sort_options)
-    ))
+    ), sort_keys=True)
```
Comment (jacquerie) [inline inspirehep/modules/search/views.py]: Testability.

**Comment by jacquerie** [comment]
I cherry-picked @jmartinm's commit from #1024, but I also modified it slightly to correct it. Will merge tomorrow by lunch.

**Pull request merged**